src/States/atDanceGameState.js

```diff
@@ -28,11 +37,25 @@ function AtDanceGame() {
 		game.load.image('enemyImg', 'assets/ADG_couple.png')
 		game.load.image('playerImg', 'assets/ADG_player.png');
 
+		game.load.image('itemImg1', 'assets/ADG_player.png');
+		game.load.image('itemImg2', 'assets/ADG_player.png');
+		game.load.image('itemImg3', 'assets/ADG_player.png');
+
+		//item dialogue
+		game.load.image('itemDialogue1', 'assets/ADG_couple.png');
+		game.load.image('itemDialogue2', 'assets/ADG_couple.png');
+		game.load.image('itemDialogue3', 'assets/ADG_couple.png');
+
+
 	}
 
 	console.log("loaded assets");
 
 	function create() {
+
+		game.physics.startSystem(Phaser.Physics.ARCADE);
+		
```

Comment:
:space_invader: Basically just one new line between `game.physics.startSystem(Phaser.Physics.ARCADE);` and `//draws tables`